src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -95,17 +98,43 @@ export class ResultsViewPageStore {
         ],
         invoke: () => {
             const clinicalDataSingleStudyFilter = {attributeIds: ["CANCER_TYPE", "CANCER_TYPE_DETAILED"], ids: this.sampleIds.result};
-            return fetchClinicalDataInStudy(this.studyId, clinicalDataSingleStudyFilter)
+            return fetchClinicalDataInStudy(this.studyId, clinicalDataSingleStudyFilter, 'SAMPLE')
         }
     }, []);
 
+    readonly clinicalDataForPatients = remoteData({
+        await: () => [
+            this.patientIds
+        ],
+        invoke: () => {
+            const clinicalDataSingleStudyFilter = {attributeIds: ["12_245_PARTC_CONSENTED"], ids: this.patientIds.result};
+            return fetchClinicalDataInStudy(this.studyId, clinicalDataSingleStudyFilter, 'PATIENT')
+        }
+    }, []);
+
+    readonly mskImpactGermlineConsentedPatientIds = remoteData({
```

Comment:
it seems like clinicalDataForPatients is already specific to "consented" because of the filter that is passed.  So I would rename the above and just do this transformation in the response (unless you're trying to reuse the call)